meeting-notes/2018/2018-06-25--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,157 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 June 25, 2018
+
+- **Lead:** Alan Shaw (@alanshaw)
+- **Notetaker:**
+- **Attendees:**
+  - @jacobheun
+  - @hugomrdias
+  - @fsdiogo
+  - @Mr0grog
+  - @achingbrain
+  - @vasco-santos
+  - @gar
+  - @vmx
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+- End of quarter actions
+  - Retrospective doc
+  - Scores for Q2 OKRs
+  - What are your objectives for next quarter?
+
+
+## Notes
+
+### Week Update
+
+@alanshaw
+  - Done:
+    - Investigated and raised an issue on go-ipfs for peer discovery regression on Windows [#5146](https://github.com/ipfs/go-ipfs/issues/5146)
+    - Added a WIP PR to expose libp2p connection manager config [#1410](https://github.com/ipfs/js-ipfs/pull/1410)
+    - Released new `js-ipfs-api` with fix for block get with empty blocks [#789](https://github.com/ipfs/js-ipfs-api/pull/789)
+    - Opened PR to address same block get issues in `js-ipfs` [#1408](https://github.com/ipfs/js-ipfs/pull/1408)
+    - Reviewed tweaked and merged PR for enabling pin tests [#1405](https://github.com/ipfs/js-ipfs/pull/1405)
+    - ...and more PR reviews and merges
+  - Blocked:
+    - `null`
+  - Next:
+    - Finish off remaining tasks for modular interface-ipfs-core tests [#290](https://github.com/ipfs/interface-ipfs-core/pull/290)
+    - Finish off remaining tasks for 0.30 release
+      - Connection manager merged ([#184](https://github.com/libp2p/js-libp2p/pull/184)) just needs a libp2p version bump
+      - By special request review and merge libp2p config PR [#166](https://github.com/libp2p/js-libp2p/pull/166)
+    - Review and merge MFS [#1360](https://github.com/ipfs/js-ipfs/pull/1360)
+
+@vmx
+ - Done:
+   - Get a full file via GraphSync hacks
+   - Review of IPLD Format changes implementation from @richardschneider
+ - Blocked: No
+ - Next:
+   - More reviews of IPLD Format changes implementation from @richardschneider
+   - Writing down what the essential core of GraphSync is from my perspective
+   - Playing around with js-libp2p with GraphSync in mind
+
+@hugomrdias
+ - Done:
+   - Implemented non-leveldb indexed db datastore
+   - secio/crypto testing
+ - Blocked:
+ - Next:
+   - Q2 retro and Q3 planning
+   - make web crypto work with local daemon
```

Comment:
What is this goal? Any discussion about it?

Reply:
@hugomrdias mind clarifying?

Reply:
I was trying to fetch blocks faster, using web crypto at least for aes but it gets stuck in the secio handshake and timeouts.

Reply:
@hugomrdias you can always run your tests with secio disabled and see what's the perf there.